resources/public/js/vendor/disqus.js

```diff
@@ -0,0 +1,23 @@
+var disqus_shortname = 'owlet';
```

Comment:
i think this line needs to be 
`var disqus_shortname = 'owlet-2';`
based on this: https://owlet-2.disqus.com/admin/settings/universalcode/